src/validateAdapter.js

```diff
@@ -0,0 +1,22 @@
+import EnzymeAdapter from './adapters/EnzymeAdapter';
+
+export default function validateAdapter(adapter) {
+  if (!adapter) {
+    throw new Error(`
+      Enzyme Internal Error: Enzyme expects an adapter to be configured, but found none. To
+      configure an adapter, you should call \`Enzyme.configure({ adapter: new Adapter() })\`
+      before using any of Enzyme's top level APIs, where \`Adapter\` is the adapter
+      corresponding to the library currently being tested. For example:
+
+      import Adapter from 'enzyme-adapter-react-15';
+
+      To find out more about this, see http://airbnb.io/enzyme/docs/installation/index.html
+    `);
+  }
+  if (!(adapter instanceof EnzymeAdapter)) {
+    throw new Error(
+      'Enzyme Internal Error: configured enzyme adapter did not inherit from the ' +
+      'EnzymeAdapter base class',
```

Comment:
lol only if your screen is set to be super narrow :-p either way our style guide allows for 100-char lines, so this string should be fine all on one line